src/annotator-tools/QuickAnnotationTool/QuickAnnotationTool.ts

```diff
@@ -21,22 +21,24 @@ export class QuickAnnotationTool extends AnnotationTool {
   initializeSuperpixels(regionSize: number) {
     if (this.throttleTimer) return;
     this.throttleTimer = true;
-    setTimeout(() => {
-      this.regionSize = Math.round(regionSize);
 
-      const superpixels = this.computeSuperpixels();
+    this.regionSize = Math.round(regionSize);
+
+    const superpixels = this.computeSuperpixels();
 
-      if (!superpixels.length) return;
+    if (!superpixels.length) return;
 
-      this.superpixels = superpixels;
-      this.superpixelsMap = {};
+    this.superpixels = superpixels;
+    this.superpixelsMap = {};
 
-      superpixels.forEach((pixel: number, index: number) => {
-        if (!(pixel in this.superpixelsMap!)) {
-          this.superpixelsMap![pixel] = [];
-        }
-        this.superpixelsMap![pixel].push(index);
-      });
+    superpixels.forEach((pixel: number, index: number) => {
+      if (!(pixel in this.superpixelsMap!)) {
+        this.superpixelsMap![pixel] = [];
+      }
+      this.superpixelsMap![pixel].push(index);
+    });
+
+    setTimeout(() => {
```

Comment:
@Andrea-Papaleo The QuickAnnotationTool test was failing after #410. It was because the test calls `initializeSuperpixels` and immediately checks the object for its state. With #410, the state isn't set until after the timeout completes, and the test doesn't wait for that timeout. I could have written a timeout in the test, but instead it made more sense to me to just move the state modifying stuff before the timeout begins. The next call to `initializeSuperpixels` will still return early.

Do we still have correct/expected behavior here?